client/src/components/ui/MultiSearch.tsx

```diff
@@ -49,6 +49,14 @@ export default function MultiSearch({
  */
 function defaultGetStringKey(node: unknown): string {
   if (typeof node === 'string') {
+    // BUGFIX: Detect psedeo separators and make sure they don't appear in the list when filtering items
```

Comment:
What view is this in @fuegovic (what part of the ui?) I did test this directly with a manually assigned array (as I wasn't sure how that model listing you provided worker), but I'm wondering if maybe there's a different point in the codebase I wasn't seeing. This might be in a place where the default filter (the one with the fix), isn't being applied. 

I can probably take another look tonight, I appreciate you sending over the yaml - I'll be able to trace where and how that data is coming across.

Reply:
this is in the model list from the main page, here's a wider screenshot

![image](https://github.com/danny-avila/LibreChat/assets/32828263/744f8052-eed8-4a68-a4b2-ae853616d57d)


Upon further testing I see now that it is properly filtering out the "pseudo separators" in other elements (like in the model list in the "model settings")

![image](https://github.com/danny-avila/LibreChat/assets/32828263/2db314b9-f008-440d-8394-1c327073d208)